### DISCUSSION TITLE
Multi device tracker

### DISCUSSION THREAD

**Discussion by pimalu** [OpenTracksApp/OpenTracks#1517] [Q&A]
Hi, 

I was searching for an open source project like https://www.georacing.com/portfolios/gps-tracking-solutions/, and I find this amazing project.

Does someone involved in OpenTracksApp knows some open source project like Georacing?

Congratulations and thank you in advance!

GB

**Comment by pstorch** [comment]
That's an interesting usecase. Would need an active server component with race participants registration and management of races. 
I don't know any FLOSS application for this case.

**Comment by dennisguse** [comment]
Something like this?

https://www.criticalmaps.net/

You need three things:
* an app that sends the current location for each person / device
* a server component that you can send data to and that distributes the locations to other participants 
* an app that shows the actual data (could be the same as the one that is sending the location)

Assumption: for this to work, you need some kind of network (assuming that peer to peer doesn't work as well).
If there is no network (or any other means to send location information), all of this will not work.

**Reply by pimalu** [reply]
I was thinking about very different solution than Criticalmaps. The system will need all the things you say. It doesn't need to send locations to other participants, just to the server. In cases were 4G isn´t reliable you have to use SBD (Short Brust Data) service trough satellite beacon.
I´ve been working on this project for the last year and half as side project, but I think it´s time to put some rhythm on it joying forces with more people. 
These guys have many things of this tread developed:
https://tacktracker.com/web/solution/livetrack
https://www.georacing.com/portfolios/gps-tracking-solutions/
https://github.com/jackyes/GoLiveTracking

This is OpenTracks sailing version:
https://www.sailracer.net/
It's mostly used on iPads

As I told you, tracking is a tread, bring Live Data to users (like OpenTracks) is another one, but still many more are needed to build a rope to hang on as much customers as possible.

Let me know if you are interested to get involved and we could start to delineate a roadmap.

**Reply by dennisguse** [reply]
I saw it :)
The whole thing is definitely doable.
My problem at the moment is a severe lack of time for things (and summer is slowly starting).

As a prototype I would recommend:
* the server component of critical maps
* and https://gpslogger.app/

**Reply by pimalu** [reply]
I appreciate your advice and guidance!  

I´m not a developer, I´m looking for someone to join with to found a team to build a sports related startup. Do you know someone who have enough experience to develop a mockup? 
There are two things to mock up:  a dashboard to show real time data (much simpler than OpenTracks) and a real time data monitor (using ¨the server component of critical maps and https://gpslogger.app/as¨ you stated before).
I have a functional user hardware interface prototype that I´m using for testing and feedback, but it´s time to move to the next stage of integrated software and hardware mockup.

**Reply by dennisguse** [reply]
We can ask around on social media if somebody is looking for a spare time project. 

For this, there needs to be at least some detailed description incl. goals etc. 
Moreover, such a tool might also be interesting for others (outside of sailing) like team cycling.
So, opening the user group might be helpful aka looking for a generic solution and when this is running make it more specialized.

**Comment by pstorch** [comment]
Or something like this as a start?
https://github.com/bilde2910/Hauk

**Reply by pimalu** [reply]
Yes, it could be a good start, but it needs to be combined with OpenTracksApp to show some parameters in real time.

**Reply by dennisguse** [reply]
@pimalu any proposals how to do this?

**Reply by pimalu** [reply]
Hi, 

Did you received my answer? I sent an answer here in another comment.

Best
